$NetBSD$

  * fix sed regular expressions to be POSIX BREs (change '\s' to
    '[[:space:]]', '+' to '\{1,\}') (not yet reported upstream)
  * quote $SUFFIX and $REVISION variable expansions (not yet reported
    upstream)

--- bootstrap.orig	2014-02-24 21:54:33.000000000 +0000
+++ bootstrap
@@ -32,7 +32,7 @@ do
              SUFFIX=$OPTARG
              ;;
          S)
-             SUFFIX=git`git log -1 --pretty=fuller --date=iso | grep CommitDate: | sed -E 's/^CommitDate:\s(.*)/\1/' | sed -E 's/(.*)-(.*)-(.*) (.*):(.*):(.*)\s+.*/\1\2\3\4\5\6/'`
+             SUFFIX=git`git log -1 --pretty=fuller --date=iso | grep CommitDate: | sed -E 's/^CommitDate:[[:space:]](.*)/\1/' | sed -E 's/(.*)-(.*)-(.*) (.*):(.*):(.*)[[:space:]]\{1,\}.*/\1\2\3\4\5\6/'`
              ;;
          r)
              REVISION=$OPTARG
@@ -65,14 +65,14 @@ then
     if [ -n "$SUFFIX" ]
     then
         echo Set package suffix "$SUFFIX"
-        sed 's/^define(\[PACKAGE_SUFFIX\],\s*\[\([-~]*[0-9a-zA-Z]*\)\])$/define(\[PACKAGE_SUFFIX\], \['$SUFFIX'\])/g' configure.ac > configure.ac.next
+        sed 's/^define(\[PACKAGE_SUFFIX\],[[:space:]]*\[\([-~]*[0-9a-zA-Z]*\)\])$/define(\[PACKAGE_SUFFIX\], \['"$SUFFIX"'\])/g' configure.ac > configure.ac.next
         mv configure.ac.next configure.ac
     fi
 
     if [ -n "$REVISION" ]
     then
         echo Set package revision "$REVISION"
-        sed 's/^define(\[PACKAGE_VERSION_REVISION\],\s*\[\([-~]*[0-9a-zA-Z]*\)\])$/define(\[PACKAGE_VERSION_REVISION\], \['$REVISION'\])/g' configure.ac > configure.ac.next
+        sed 's/^define(\[PACKAGE_VERSION_REVISION\],[[:space:]]*\[\([-~]*[0-9a-zA-Z]*\)\])$/define(\[PACKAGE_VERSION_REVISION\], \['"$REVISION"'\])/g' configure.ac > configure.ac.next
         mv configure.ac.next configure.ac
     fi
 fi
